### DISCUSSION TITLE
GitHub Student Pack Application Being Automaticly Rejected

### DISCUSSION THREAD

**Discussion by YuzhenQin1** [community/community#145352] [GitHub Education] [answered]
### Hi everyone,

I applied for GitHub Student Pack on Nov. 16, and it was rejected on Nov. 22. I then submitted 3 applications on Nov. 23 WITH THREE DIFFERENT PROOF FROM THREE DIFFERENT CATEGORIES, and they were automaticly rejected with the same reason:

> **What happened?**
> Suggested proof to expedite your approval:
> - Please ensure that your academic affiliation document contains your last name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). If you have only a single legal name, enter it in both the first and last name fields of your GitHub billing information.
> - Your image contains keywords indicating that it may be an acceptance letter. Only proof of current academic enrollment is acceptable.
> - Please ensure that your academic affiliation document contains your first name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). Variations of your name and nicknames are unacceptable.

1. I DO NOT have a "single legal name", there are first name and last name, separated with comma, on my passport. However, my first name only contains a single character.
2. My image DOES NOT contain keywords indicating that it may be an acceptance letter. In fact, what I uploaded was a school ID.
3. My name on the document is the same as the name appears in my GitHub billing information.

Best Regards,  
Yuzhen Qin

**Comment by 3ab3z2** [comment]
Same here too, I tried multiple times with different documents, different cameras, and even with translated documents, non of that worked, eventhough I got accepted before with the same type of document, not it won't get accepted

**Reply by soo-s** [reply]
Ensure that the name in the user information section, the name in the payment information section on GitHub, and the name in the uploaded translated student status report image are all consistent. Sometimes when the webpage translates images, the name translation might be missing, leading to consistent errors reporting inconsistent name information. Check if the name in your uploaded image is missing any translation.

**Comment by cesarbarbozarueda** [comment]
It doesn't work, it's a farce. They make you waste your time and in the end they don't approve it. I tried different cameras, and there's no way. Surely this error

**Comment by YuzhenQin1** [comment, marked as answer]
In the end I was automaticly approved ... with a proof that doesn't even have a date on it.

Seamingly GitHub have used some kind of OCR model to automate the approve/reject process.

**Reply by 3ab3z2** [reply]
Mine also just got approved today, I retried verifying it by holding the laptop itself and taking a picture of the proof and it just got accepted? lol
